server/controllers/finance/reports/financial.patient.handlebars

```diff
@@ -4,9 +4,20 @@
 
   {{> header }}
 
-  <h3 class="text-center text-uppercase"><strong>{{translate "PATIENT_RECORDS.FINANCIAL_ACTIVITY.REPORT_TITLE"}}</strong></h3>
-  <h4 class="text-center"><strong>{{ patient.reference }} | {{ patient.display_name }}</strong></h4>
-  <h4 class="text-center"><strong>{{ patient.debtor_group_name }} ({{ patient.number }})</strong></h4>
+  <h3 class="text-center text-uppercase">
+    <span style="text-decoration-line: underline">
+      <strong> {{translate "PATIENT_RECORDS.FINANCIAL_ACTIVITY.REPORT_TITLE"}} </strong>
+    </span>  
+  </h3>
+  <h4 class="text-uppercase"><strong>{{ patient.reference }} | {{ patient.display_name }}</strong></h4>
```

Comment:
What was wrong with centering these?

Reply:
They just wished that the Balance could be posted on the header of the Report as in Bhima 1X, but also that there could be distinctly the Debit and Credit Column for the Patient and the Employee financial Activity